src/AmsterdamPHP/Bundle/MeetupBundle/Command/CreateJoindinPlaceholderCommand.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace AmsterdamPHP\Bundle\MeetupBundle\Command;
+
+use CL\Slack\Payload\ChatPostMessagePayload;
+use Codeliner\ArrayReader\ArrayReader;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class CreateJoindinPlaceholderCommand
+ */
+class CreateJoindinPlaceholderCommand extends ContainerAwareCommand
+{
+    /**
+     * @var \AmsterdamPHP\Bundle\MeetupBundle\Service\EventService
+     */
+    protected $meetup;
```

Comment:
Could you elaborate more on that? I'm curious :)
In my opinion it's much easier to open things up later, than to close them down, without knowing whats (ab)using them.